storybook/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "unison-uicore-storybook",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.21.0",
+    "@storybook/addon-actions": "^6.5.16",
+    "@storybook/addon-essentials": "^6.5.16",
+    "@storybook/addon-interactions": "^6.5.16",
+    "@storybook/addon-links": "^6.5.16",
+    "@storybook/builder-webpack5": "^6.5.16",
+    "@storybook/html": "^6.5.16",
+    "@storybook/manager-webpack5": "^6.5.16",
+    "@storybook/testing-library": "^0.0.13",
+    "babel-loader": "^8.3.0",
+    "elm-webpack-loader": "^8.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  }
+}
```

Comment:
Can we remove the fields we don't need:
`version`, `description`, `main`, `scripts.test`, `keywords`, `author`.

Also change license to `MIT` please.